beacon_node/http_api/tests/tests.rs

```diff
@@ -1578,12 +1592,14 @@ impl ApiTester {
             .await
             .0;
 
-        assert!(
-            self.client
-                .post_beacon_blocks(&PublishBlockRequest::from(block))
-                .await
-                .is_err()
-        );
+        let response: Result<Response, Error> = self
+            .client
+            .post_beacon_blocks_v2(&PublishBlockRequest::from(block), None)
+            .await;
+
+        assert!(response.is_ok());
+
+        assert_eq!(response.expect("infailable").status(), StatusCode::ACCEPTED);
```

Comment:
typo, but can just unwrap:

```suggestion
        assert_eq!(response.unwrap().status(), StatusCode::ACCEPTED);
```